server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -521,6 +521,7 @@ export class AgenticChatController implements ChatHandlers {
                     session.setDeferredToolExecution(toolUse.toolUseId, deferred.resolve, deferred.reject)
                     this.#log(`Prompting for tool approval for tool: ${toolUse.name}`)
                     await deferred.promise
+                    await chatResultStream.writeResultBlock(this.#getUpdateBashConfirmResult(toolUse, true))
```

Comment:
is bash the only tool that needs confirmation right now and thats why it's safe to add that here?

Reply:
Yeah, I plan to work on fsWrite approval next, and I can refactor this. We will need different mynah ids for that one I believe.